.editorconfig

```diff
@@ -2,7 +2,7 @@ root = true
 
 [*]
 indent_style = space
-indent_size = 4
```

Comment:
that would mean reformatting the whole java stuff, right? 
does intellij respect an additional editor config in the frontend-new dir?